### PR TITLE
[EDU-4485] refactor: access Edge Storage via S3 - guide update

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-4485] 

### Changes

- Update access Edge Storage via S3 guide to include a section with commands.
- Refine title.

### Additional links

n/a.

[EDU-4485]: https://aziontech.atlassian.net/browse/EDU-4485?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx [hunk 1]
```diff
@@ -103,6 +103,26 @@ Você pode salvar suas configurações em um arquivo `.s3cfg` se necessário e m
...
+
+Depois de configurar o acesso ao bucket através do s3cmd, você pode executar operações para gerenciar seus objetos no Edge Storage usando os comandos disponíveis.
+
+| Escopo | Descrição | 
+|---|---|
+| `s3cmd ls` | Listar todos os seus buckets |
+| `s3cmd mb s3://meu-novo-nome-do-bucket` | Criar um novo bucket. Os nomes dos buckets devem ser únicos e as melhores práticas para nomear buckets incluem especificar que tipos de objetos estão armazenados e o tipo de permissões para os objetos |
+| `s3cmd ls s3://meu-novo-nome-do-bucket` | Listar o conteúdo do bucket |
+| `s3cmd put file.xml s3://meu-novo-nome-do-bucket/file.xml` | Fazer upload de um arquivo para o bucket |
+| `s3cmd get s3://meu-novo-nome-do-bucket/file.xml file-2.xml` |  Recuperar o arquivo e verificar se ele não foi corrompido |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx:116]:
```suggestion
| `s3cmd get s3://meu-novo-nome-do-bucket/file.xml file-2.xml` | Recuperar o arquivo e verificar se ele não foi corrompido |
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx [hunk 2]
```diff
@@ -103,6 +103,26 @@ Você pode salvar suas configurações em um arquivo `.s3cfg` se necessário e m
...
+| `s3cmd del s3://meu-novo-nome-do-bucket/addrbook.xml s3://meu-novo-nome-do-bucket/storage.jpg` |  Excluir o objeto |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/use-s3-protocol/index.mdx:117]:
```suggestion
| `s3cmd del s3://meu-novo-nome-do-bucket/addrbook.xml s3://meu-novo-nome-do-bucket/storage.jpg` | Excluir o objeto |
```

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/en/pages/guides/edge-storage/use-s3-protocol/index.mdx [hunk 3]
```diff
@@ -104,6 +104,26 @@ You can save your settings into an `.s3cfg` file if needed and modify them again
...
+
+| Scope | Description | 
+|---|---|
+| `s3cmd ls` | List all your buckets |
+| `s3cmd mb s3://my-new-bucket-name` | Create a new bucket. Bucket names must be unique and best practices for naming buckets include specifying what types of objects are stored and the type of permissions for the objects |
+| `s3cmd ls s3://my-new-bucket-name` | List the contents of the bucket |
+| `s3cmd put file.xml s3://my-new-bucket-name/file.xml` | Upload a file into the bucket |
+| `​​s3cmd get s3://my-new-bucket-name/file.xml file-2.xml` | Retrieve the file back and verify that it hasn't been corrupted |
+| `s3cmd del s3://my-new-bucket-name/addrbook.xml s3://my-new-bucket-name/storage.jpg` | Delete the object |
+| `s3cmd rb s3://logix.cz-test` | Remove the bucket. Only empty buckets can be removed |
```
Comment (luizcunhaazion) [inline src/content/docs/en/pages/guides/edge-storage/use-s3-protocol/index.mdx:119]: It is important to inform the user, that files deleted within the past 24 hours will block the deletion of the entire bucket.

Comment (MarianaAguilera) [reply src/content/docs/en/pages/guides/edge-storage/use-s3-protocol/index.mdx:119]: A note was added in this section.

**Review by luizcunhaazion — APPROVED** [review]

**Pull request merged**